examples/live/run.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+# example/test of running PyCBC Live on simulated data
+
+set -e
+
+export OMP_NUM_THREADS=4
+export HDF5_USE_FILE_LOCKING="FALSE"
+
+gps_start_time=1272790000
+gps_end_time=1272790500
+
+echo -e "\\n\\n>> [`date`] Generating simulated strain"
+
+function simulate_strain { # detector PSD_model random_seed
+    mkdir -p strain/$1
+    pycbc_condition_strain \
+        --fake-strain $2 \
+        --fake-strain-seed $3 \
+        --output-strain-file "strain/$1/$1-SIMULATED_STRAIN-{start}-{duration}.gwf" \
+        --gps-start-time $gps_start_time \
+        --gps-end-time $gps_end_time \
+        --sample-rate 16384 \
+        --low-frequency-cutoff 10 \
+        --channel-name $1:SIMULATED_STRAIN \
+        --frame-duration 32
+}
+
+simulate_strain H1 aLIGOMidLowSensitivityP1200087 1234
+simulate_strain L1 aLIGOMidLowSensitivityP1200087 2345
+simulate_strain V1 AdVEarlyLowSensitivityP1200087 3456
+
+echo -e "\\n\\n>> [`date`] Running PyCBC Live"
+
+mpirun \
```

Comment:
I think this is for the intel mpirun api. We'll need to see what Travis supports in this regard. Most likely openmpi  or mpich

Reply:
It is actually for Open MPI. We are already installing some MPI version on Travis in order to run the `--help` test, but yes, we will see if this example works with that.